.gitattributes

```diff
@@ -0,0 +1,26 @@
+* text=auto
+
+*.css eol=lf
+*.eslintrc eol=lf
+*.gitattributes eol=lf
+*.gitignore eol=lf
+*.gradle eol=lf
+*.h eol=lf
+*.html eol=lf
+*.java eol=lf
+*.js eol=lf
+*.jscsrc eol=lf
+*.json eol=lf
+*.m eol=lf
+*.map eol=lf
+*.md eol=lf
+*.modulemap eol=lf
+*.npmignore eol=lf
+*.pch eol=lf
+*.plist eol=lf
+*.sh eol=lf
+*.xml eol=lf
+*.yml eol=lf
+
+*.jar binary
+*.png binary
```

Comment:
found all extensions used in this repo with 
`git ls-tree -r HEAD --name-only | perl -ne "print \$1 if m/\.([^.\/]+)$/" | sort -u`

I listed them explicitly because with `* text=auto` git on Windows is not always smart enough and needs some guidance